### DISCUSSION TITLE
v1.17.0

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#724] [General]
See [changelog](https://github.com/th-ch/youtube-music/blob/master/changelog.md#v1170) for the list of updates and the full diff.

Thanks to all contributors! 🏅

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.17.0'>v1.17.0</a>.</em>

**Comment by sanctifyy** [comment]
Are there any plans to introduce themes like spicetify?

**Comment by supramacy** [comment]
i just wanted to congratulate you for such an awesome job, saved my life, thx man!

**Comment by therohanrathee** [comment]
The Album Art doesn't gets downloaded with the newer versions

**Comment by pemot** [comment]
2nd time Ive turned latest version on and youtube immediately inserts an ad. Can you update the software - its great to be ad free and really appreciate that you are offering this. Thx in advance

**Reply by Araxeus** [reply]
Correct me if I'm wrong but it only happens right after launch on the first song right?

It seems somehow the adblocker doesn't load soon enough

**Comment by pemot** [comment]
Yes, exactly! But both these 2 times only happened in the last week. Google
has never progressed to a better revenue model than being an Advertising
company LOL

ᐧ

**Comment by crazyrokr** [comment]
Transparency of menu stopped working in 1.17

**Reply by Araxeus** [reply]
what do you mean? could you provide a screenshot comparison?

**Reply by crazyrokr** [reply]
As you can see in the following screenshots, version 1.16 has transparent menus, while version 1.17 does not.

1.16
![1 16](https://user-images.githubusercontent.com/56276969/179531470-166d4dfc-ca13-4235-994d-7f292d42bc1f.png)

1.17
![1 17](https://user-images.githubusercontent.com/56276969/179531510-96ff9a42-4365-4419-94b4-2ce5508668e8.png)

**Reply by Araxeus** [reply]
we didn't change that, probably something that changed when updating the [custom-electron-titlebar](https://github.com/AlexTorresSk/custom-electron-titlebar) dependency (which enables this menu)

I'll try to look into it

**Comment by he4dch4rge** [comment]
Downloading option it's not working v17. 
![downloading playlist not working](https://user-images.githubusercontent.com/111712381/185815524-ba57b7b4-8bab-4a21-af3a-b8d7f92bbd69.png)
